Ginger/Ginger/RunSetPageLib/RunsetOperationsPage.xaml.cs

```diff
@@ -90,12 +90,29 @@ public void Init(RunSetConfig runSetConfig)
             SetContentAndEventsListeners();
         }
 
+        RunSetActionEditPage runSetActionEditPage;
         private void RunSetActionsGrid_RowChangedEvent(object sender, EventArgs e)
         {
             RunSetActionEditFrame.Content = null;
             if (RunSetActionsGrid.CurrentItem != null)
             {
-                RunSetActionEditPage RSAEP = new RunSetActionEditPage((RunSetActionBase)RunSetActionsGrid.CurrentItem);
+                RunSetActionEditPage RSAEP = null;
+                if (((Amdocs.Ginger.Repository.RepositoryItemBase)RunSetActionsGrid.CurrentItem).ItemName == "Open ALM Defects")
```

Comment:
Why specific handling needed for Defects Operation. We have other runset operations like Send email and report etc. 
So either this issue exist also for those pages. Or we need some specific fix for Defects Operation in that page.

Reply:
Here what is happening that every time the operation is selected from the grid then it again instantiating the component page and every time the newly created page instance is displayed, now for rest of operations it is also behaving the same way but i.e. again creating the instance of selected operations page there it is not messing the data already selected but here in this case "GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(RadioDefectsOpeningModeForAll, RadioButton.IsCheckedProperty, runSetActionAutomatedALMDefects, nameof(RunSetActionAutomatedALMDefects.DefectsOpeningModeForAll));" the binding used for the radio button is not working correctly when the page is loaded 2nd and after that. when this binding is called for the radio button it resetting all selections for that control.

Also why to recreate the page again and again when we have its instance then it should be used to load it?